src/display/api.js

```diff
@@ -1231,8 +1232,8 @@ var PDFWorker = (function PDFWorkerClosure() {
     // pdf.worker.js file is needed.
     if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) {
       if (typeof SystemJS === 'object') {
-        Promise.all([SystemJS.import('pdfjs/core/network'),
-                     SystemJS.import('pdfjs/core/worker')]).then((modules) => {
+        Promise.resolve(SystemJS.import('pdfjs/core/worker')).
+          then((modules) => {
```

Comment:
Nit: Please indent *this* line with two more spaces, and also change it to `then(worker) => {`.